.gitignore

```diff
@@ -5,3 +5,6 @@ npm-debug.log
 /gh-tmp
 .DS_Store
 .idea
+/dist/lib
+/dist/util
+/dist/test
```

Comment:
Taken care of in https://github.com/Automattic/wpcom.js/commit/4a1cb85c54a3f55e961068d4479ebfba0e589314